build_secrets.py

```diff
@@ -103,59 +85,49 @@ def build_package():
     return
 
 
-class PostInst():
+def clean_login(login):
```

Comment:
This function does not look necessary.
```
USER=robin script.py
```
and
```
USER='robin' script.py
```
Is exactly the same thing. Even with double quotes.